gluon-client-bridge: add WPA3 Enhanced Open client radio support (#1945)
freifunk-gluon/gluon: docs/features/private-wlan.rst

It is possible to set up a private WLAN that bridges the WAN port and is separated from the mesh network.
Please note that you should not enable ``mesh_on_wan`` simultaneously.

> The private WLAN is encrypted via WPA2 by default. On devices with enough flash and a supported radio,

Reviewer: ```suggestion
The private WLAN is encrypted through WPA2 by default. On devices with enough flash and a supported radio,
```

WPA3 or WPA2/WPA3 mixed-mode can be used instead of WPA2. For this to work, the ``wireless-encryption-wpa3``

> feature has to be added to the list of your supported features.

Reviewer: You are talking about `GLUON_FEATURES` here? If so add the variable name explicitly.

Additionally, IEEE 802.11w management frame protection should be enforced for WPA2/WPA3, however this

> can lead to connectivity problems for older clients. In this case, management frame protection can be

Reviewer: ```suggestion
can lead to connectivity problems for older clients. Therefore management frame protection can be
```

> configured to be optional or disabled in the advanced settings tab.

Reviewer: ```suggestion
made be optional or completely disabled in the advanced settings tab.
```

> set to optional or disabled completely using advanced settings. 

Reviewer: can be configured to be optional or disabled in advanced settings tab.
